app/src/androidTest/java/com/duckduckgo/app/browser/BrowserTabViewModelTest.kt

```diff
@@ -118,7 +115,12 @@ import com.duckduckgo.app.trackerdetection.model.TrackingEvent
 import com.duckduckgo.app.usage.search.SearchCountDao
 import com.duckduckgo.app.widget.ui.WidgetCapabilities
 import com.duckduckgo.feature.toggles.api.FeatureToggle
-import com.duckduckgo.privacy.config.api.*
+import com.duckduckgo.privacy.config.api.AmpLinkInfo
```

Comment:
Have a look at your IDE settings, you have a lot imports that should not be changing and are adding unnecessary changes to the PR.

Reply:
This is probably because I have "Optimize imports" checked so it removes all unused imports. It shouldn't break anything as it only removes imports that we are not using anymore or it will replace * to the specific class so it is lighter. Did you have problems with this setting before?

Reply:
You don't need to mark optimize imports (or anything really) because Spotless already does it for you, same with any other formatting stuff. Did you add the pre-commit hook?

Reply:
Yes, I'm also so used to do control+option+O when I delete things so imports that are not used anymore get removed. I normally also check "Optimize imports" while committing in the IDE. I think I need to check how you use Spotless so I am in the same page as you guys.